WIP: (GH-57) Cleanup Secret* modules (#60)
MicrosoftDocs/PowerShell-Docs-Modules: reference/ps-modules/Microsoft.PowerShell.SecretManagement/Set-Secret.md

## DESCRIPTION

This cmdlet adds a secret value by name to SecretManagement. If no vault name is specified, then the

> secret is added to the default vault. If an existing secret by the same name exists, it is

Author: ```suggestion
secret is added to the default vault. If secret with that name exists, it is overwritten.
```

### -SecretInfo

Specifies a **SecretInformation** object describing a stored secret returned by `Get-SecretInfo`.

> This enables moving secrets from one extension vault to another.

Author: ```suggestion
This enables copying secrets from one extension vault to another.
```
